Updating README with TinaCMS dev badge (#4612)
tinacms/tinacms: _docs/dev-badge.md

## 🧑‍🎓 TinaCMS Dev Badge

> We encourage all developers to get their “TinaCMS badge” - that is, to get it up and running and complete the basics before implementing TinaCMS on a real project. This is a must to prove that you can use TinaCMS!

Reviewer: ```suggestion
We encourage all developers to earn their “TinaCMS badge” by getting TinaCMS up and running and complete the basics before implementing TinaCMS on a real project. This is essential to prove that you can effectively use TinaCMS!
```

Here are the steps to complete your journey:

1. Join [Tina's Discord channel](https://discord.com/invite/zumN63Ybpf) - You can leave a message on the `#introductions`, i.e.:

   > Hi, I am trying out TinaCMS with {{ TECH eg. NextJS }}
     After I have evaluated TinaCMS, I plan to use it in the {{ INDUSTRY }} industry.  
     Previous CMSs I have used include {{ FILL IN }}

2. Get TinaCMS running and deploy a site using [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin)
   
   Feel free to start with one of the starter templates

>    This can be a personal site, or one you did for work

Reviewer: ```suggestion
   This can be a personal site or one you’ve built for work
```

3. Record a [Done Video](https://www.ssw.com.au/rules/record-a-quick-and-dirty-done-video/) explaining how Tina works

>    What we are looking for:

Reviewer: ```suggestion
   Here's what we're looking for:
```

   - Who you are
   - Why you are using TinaCMS
   - Something you like about TinaCMS
   - Any feedback or suggestions you have from your initial tryout
     Tip: Have each suggestion's discussion or issue open in a separate tab

> 4. Post the vidoe to YouTube (or other video platform) - make sure that it is public

Reviewer: ```suggestion
4. Post the video to YouTube (or another video platform) - make sure that it is public
```

5. (Optional) Post your video on social media (e.g., LinkedIn, Discord, X...)

> 6. Let us know about it! Email badge@tina.io

Reviewer: ```suggestion
6. Let us know about it! Email us at badge@tina.io
```

>    Include a link to:

Reviewer: ```suggestion
   Include links to:
```

>    - your video

Reviewer: ```suggestion
   - Your video
```

>    - links to your posts

Reviewer: ```suggestion
   - Your social media posts (if applicable)
```
